app/models/invoice/data_presenter.rb

```diff
@@ -49,9 +50,9 @@ def order_completed_at
     end
 
     def checkout_adjustments(exclude: [], reject_zero_amount: true)
-      adjustments = all_eligible_adjustments
+      adjustments = all_eligible_adjustments.map(&:clone)
```

Comment:
Why was this needed?

Reply:
```ruby
      adjustments = all_eligible_adjustments.map(&:clone)

      adjustments.reject! { |a| a.originator.type == 'Spree::TaxRate' }
```
without `.map(&:clone)`, calling `reject!` on `adjustments` will affect `all_eligible_adjustments` too